rfc-015-small-group-comradro.md

```diff
@@ -0,0 +1,23 @@
+# Organise Small Group Comradros
+
+## Summary
+
+Propose and join fortnightly small group retrospectives/workshops/discussions/other on topics of your choosing.
+
+## Problem
+
+There is no organisation currently about how Small Group Comradros should be organised, by who, how to join them, where to keep track of them, and when they should be run.
+
+## Proposal
+
+Anyone interested in proposing a Small Group Comradro **SHOULD** post their idea as a ticket in the Upcoming column of [the Trello Board][sgc-trello] by lunchtime Friday on a Comradro week, assigning themselves to it.
+
+Anyone interested in joining a Small Group Comradro **SHOULD** assign themselves to **ONE** proposal they're interested in by lunchtime Friday on a Comradro week.
+
+Proposals **SHOULD** be mentioned at the start of Learn Tech by each team, as people get used to the new format.
```

Comment:
Is this inclusive to nontech part of Made Tech ?

Reply:
No. I guess really people should know to check the Slack, expected this to be a short term reminder more than anything. Slack post as well?

Reply:
Could be generic in this RFC to get things rolling?

> Proposals **MUST** be made public (e.g. slack post, mentioning at QT/learn tech)